test/integration/triggered_boundingbox_camera.cc

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
```

Comment:
```suggestion
 * Copyright (C) 2023 Open Source Robotics Foundation
```

Reply:
Addressed in [d7bc027](https://github.com/gazebosim/gz-sensors/pull/322/commits/d7bc027df336296e9e27c52272e98b4154faf691)